### ISSUE TITLE
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: AVERAGE_POOL_2D, CONCATENATION, CONV_2D, FULLY_CONNECTED, MAX_POOL_2D, MEAN, RELU, SOFTMAX, STRIDED_SLICE. Here is a list of operators for which you will need custom implementations: AddV2.

### ISSUE THREAD

**Issue by ammu11** [tensorflow/tensorflow#33393]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installed from (source or binary):
- TensorFlow version (or github SHA if from source):

**Provide the text output from tflite_convert**

```
# Copy and paste here
```

Also, please include a link to a GraphDef or the model if possible.

**Any other info / logs**

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by gadagashwini** [comment]
@ammu11, Please Provide the exact sequence of commands / steps that you executed before running into the problem. And also provide the Tensorflow version. Thanks!

**Comment by ammu11** [comment]
While converting pb file to tflite, facing this issue. And my tensorflow
version is 1.14.0. And here is the tflite command which I have used
-> tflite_convert --output_file=D:/Images/car-damage-dataset/fnp.tflite
--graph_def_file=D:/Images/car-damage-dataset/fnp.pb
--input_shapes=1,224,224,3 --input_arrays=input_1 --output_arrays=output_0
--mean_values=0 --std_dev_values=128 --allow_custom_ops

**Comment by haozha111** [comment]
Could you paste your .pb file here so that i can give it a try?

**Comment by andreydung** [comment]
I have the exactly the same problem. Running the sample code from doc:
```
import tensorflow as tf

model = tf.keras.applications.MobileNetV2(
    weights="imagenet", input_shape=(224, 224, 3))

# Convert the model.
converter = tf.lite.TFLiteConverter.from_keras_model(model)
tflite_model = converter.convert()
```
I used tf2.0, tf2.0-rc1 but these operations are still not found:
```
Here is a list of operators for which you will need custom implementations: AddV2, FusedBatchNormV3.
```

**Comment by ammu11** [comment]
Thanks for your reply.

I have updated the tensorflow version to 1.15.0 and worked for me.
